<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_chart</class>
 <widget class="QWidget" name="Form_chart">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="legendComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="animatedLabel">
       <property name="text">
        <string>Animation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>Theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="themeComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="legendLabel">
       <property name="text">
        <string>Legend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="animatedComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>328</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="antialiasCheckBox">
       <property name="text">
        <string>Anti-aliasing</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
